docs/api-docs/apps/guide/apps-04-devloper-portal.md

```diff
@@ -1,87 +1,77 @@
+Create, edit, and submit apps for approval using the [Developer Portal](https://devtools.bigcommerce.com/). In [Beginning App Development](/api-docs/apps/guide/development) we briefly touched on how to create a draft app. In this article, we'll go over how to perform other common app management tasks. To get started, sign in or create an account in the [Developer Portal](https://devtools.bigcommerce.com/).  
 
 <!-- theme: info -->
-> #### Note
-> * `DRAFT` apps can only be installed on stores owned by the same email address as the developer portal account's email address.
+> #### Store email address constraint
+> Apps that aren't approved for distribution through the [Apps Marketplace](https://bigcommerce.com/apps) can only be installed on stores owned by the same email address as the developer portal account's email address.
 
+## Create an app
 
+To create an app, sign in to or create an account with the [Developer Portal](https://devtools.bigcommerce.com). Creating an app also creates an app API account. To learn more about app API accounts, see the [Guide to API Accounts](/api-docs/getting-started/authentication/rest-api-authentication#app-api-accounts).
 
-## Creating apps
+1. Click the **Create an app** button on the right side of the landing page.
 
-To create an app, do the following:
+![Create an App](https://storage.googleapis.com/bigcommerce-production-dev-center/images/apps-04-developer-portal-01.png "Create an App")
 
-1. Navigate to **[My Apps](https://devtools.bigcommerce.com/my/apps)**.
-2. Click **Create an app**.
-3. Give the app a name.
-4. Click **Create**.
+2. Give your app a name. This name is only visible to you.
+3. Click **Create**.
+4. At the top of the dialog that opens next, click the **Technical** tab. Scroll down to assign your app the desired OAuth scopes. 
 
-![Developer Portal](https://storage.googleapis.com/bigcommerce-production-dev-center/images/apps-04-developer-portal-01.png "Developer Portal")
+![Technical](https://storage.googleapis.com/bigcommerce-production-dev-center/images/app-api-account/devtools-technical.png "Technical")
 
-[Learn more about completing registration fields and submitting apps for approval](/api-docs/apps/guide/publishing).
+![Assign OAuth scopes](https://storage.googleapis.com/bigcommerce-production-dev-center/images/app-api-account/devtool-oauth-scopes.png "Assign OAuth scopes")
 
-<!-- theme: info -->
-> #### Note
-> * The app's name is only visible in the developer portal.
+5. Click the **Update & Close** button on the dialog's lower right.
 
+<!-- theme: info -->
+> #### Information optional
+> When you create or edit an app in the Dev Portal, no app information fields are mandatory unless you're preparing the app for BigCommerce [Apps Marketplace](https://bigcommerce.com/apps) approval. To learn more about completing an app profile for approval, see our [App Publishing Guide](/api-docs/apps/guide/publishing).

```

Comment:
```suggestion
> No app information fields are mandatory when you create or edit an app in the Dev Portal unless you're preparing the app for BigCommerce [Apps Marketplace](https://bigcommerce.com/apps) approval. To learn more about completing an app profile for approval, see our [App Publishing Guide](/api-docs/apps/guide/publishing).

```

Reply:
I reworked this in a forthcoming push. I started with your suggestion, but it ended up a little different.